app/controllers/authorize_interactions_controller.rb

```diff
@@ -38,7 +38,7 @@ def set_resource
 
   def located_resource
     if uri_param_is_url?
-      ResolveURLService.new.call(uri_param)
+      ResolveUrlService.new.call(uri_param)
```

Comment:
`inflect.acronym 'URL'`?

Reply:
It's a possibility, but we also have classes using `Url`, for some reason I thought it was easier this way.

Reply:
Added a commit adding the `URL` inflection and changing class names accordingly.

Reply:
awesome! I agree that URL is probably the best to standardize on.